### DISCUSSION TITLE
Newly Built Boards Extreamly Unstable

### DISCUSSION THREAD

**Discussion by Babelfisher1** [erichelgeson/BlueSCSI-v1#229] [Q&A]
Wondering if anyone has had troubles with these Bluepills? 
I purchased a pack of 5 bluepills from ebay that looked at the time of purchase to be compatible. 
They have the following writing on them:

**STM32
F103C8T6 
991kA 93 
MYS 806**                           
https://freeimage.host/i/thumbnail-img-7016.Hk4VGN2 (See close up pic of the chip)

The 5 completed boards are all incredibly unstable. Occasionally booting but most of the time showing a happy mac then a random system error message.

I've tried 4 different SD cards that have all been put through a full wipe with the recommended formatting tool and formatted to exfat.  As well as several different HDD images with a few directly downloaded form macintoshrepository.org......Nothing works.

I've put one of the flashed bluepills on a know working board purchased last year and it performs exactly the same unstable way. 
On top of all that I also put the my working Bluescsi's bluepill on one of my new boards and it works perfectly.

After a lot of testing and changing parts around I think the problem must be this particular iteration of knockoff bluepill that is causing the instability. 

The flashing process seems to work without errors and verifies with checksums etc.  Wondering if these bluepills could be accepting the firmware but then causing the instability?

**Comment by erichelgeson** [comment]
> After a lot of testing and changing parts around I think the problem must be this particular iteration of knockoff bluepill that is causing the instability.

> The flashing process seems to work without errors and verifies with checksums etc. Wondering if these bluepills could be accepting the firmware but then causing the instability?

Defiantly could be, not all are created equal. A few things I've done to recover bluepills that behave somewhat like your description:

* Check the SMT32 chip and all legs on the chip. I've seen cold solder joints where a pin just doesn't make enough continuity. 
* Check the voltage at the 3.3 and 5v - have seen bad voltage regulators on some.
* Power it via USB (also to ensure good voltage)
* Try a different SD card, preferably a different brand - WCH clones have issues with Sandisk.

If those don't resolve, you likely got a bad batch. Hopefully it's in your return window. It's been a while since I've bought a batch, but have usually had good results from Ali.

> As well as several different HDD images with a few directly downloaded form macintoshrepository.org

We don't post or test any images from that site, check out the Usage page for links to known working image (though it sounds like you have one)
